README.md

```diff
@@ -19,3 +19,58 @@ from hydra_openapi_parser import openapi_parser
 ```
 
 Porting out from hydrus the hydraspecs directory
+
+## Sample use-cases of openapi_parser module
+
+Once the OpenAPI YAML document has been loaded into the program a python dictionary `doc`, you can do the following:
```

Comment:
This is badly expressed. Please correct to: "Once the OpenAPI YAML document has been assigned into the variable `doc` as a Python dictionary"